be/src/services/entity.service.ts

```diff
@@ -107,11 +107,27 @@ export class EntityService {
     });
   }
 
-  // TODO remove rule when implemented
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  async deleteEntity(vocabID: string, entityID: string, lastModified: string): Promise<unknown> {
-    return Promise.resolve(null);
-  }
+    // TODO remove rule when implemented
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    public async deleteEntity(vocabID: string, entityID: string, lastModified: Date): Promise<boolean> {
+
+        if (!await vocabularyService.getVocabular(vocabID)) {
+            throw new KnowledgeError(404, 'Vocabulary', 'No vocabulary matches the provided ID.')
+        }
+
+        if (!await this.getEntity(vocabID, entityID)) {
+            throw new KnowledgeError(404, 'Entity', 'No entity matches the provided ID.')
+        }
+
+        return EntityService.collection().deleteOne({_id: new ObjectId(entityID), lastModified: lastModified})
+            .then(r => {
+                if (r.deletedCount == 1) {
```

Comment:
```suggestion
                if (r.deletedCount === 1) {
```
See https://www.jstips.co/en/javascript/use_===_instead_of_==/, as `===` is more strict and type save